README.md

```diff
@@ -1,11 +1,11 @@
-# Sopel GitHub
+# sopel-github
 
 GitHub plugin for [Sopel](https://sopel.chat/) IRC bots.
 
 
 ## Installation
 
-The easy (and recommended) way: `pip install sopel_modules.github`
+The easy (and recommended) way: `pip install sopel-github`
 
 The less-easy way; you must already have Sopel installed to use this method.
```

Comment:
Two things for my future self, when he's at a computer:

1. Don't advertise the source installation (here)
2. Fix the sample command to install with the `emoji` extra (below)